doc/api/buffer.markdown

```diff
@@ -1358,7 +1636,7 @@ const store = [];
 socket.on('readable', () => {
   var data = socket.read();
   // allocate for retained data
-  var sb = new SlowBuffer(10);
+  var sb = SlowBuffer(10);
```

Comment:
I know this is just a backport so perhaps it should be fixed elsewhere, but the `SlowBuffer` documentation is a bit strange. The documentation says that `SlowBuffer` is a class (a few lines above) and should be called with `new` (a few lines below), but this change removes the `new` keyword from the example code.